sdk/storage/azure-storage-blobs/test/blob_container_client_test.cpp

```diff
@@ -732,7 +732,7 @@ namespace Azure { namespace Storage { namespace Test {
     EXPECT_NO_THROW(containerClient2.GetProperties());
   }
 
-  TEST_F(BlobContainerClientTest, Tags)
+  TEST_F(BlobContainerClientTest, DISABLED_Tags)
   {
```

Comment:
Why?
Wasn't it working already?

Reply:
it's still in public preview. It works in some regions, while doesn't work in others.